rs/replicated_state/src/canister_state/queues/tests.rs

```diff
@@ -74,6 +74,20 @@ impl CanisterQueuesFixture {
         )
     }
 
+    fn try_push_deadline_expired_input(&mut self) -> Result<(), String> {
+        self.last_callback_id += 1;
```

Comment:
Absolutely not for this PR, but conceptually there should be one callback ID for every successful `push_output_request`. If you're going in the direction of bundling the `CallContextManager` together with the queues, I do wonder if we have any practical chance of actually enforcing this, i.e., somehow have `CallbackId` be only generated through `push_output_request`. I realize you'd need to thread some canister-global state through to keep the numbers unique across the different queues.  And I now also realize that there are two `next_callback_id` fields, one in `SandboxSafeSystemState`, and one in `CallContextManager`, and I have no idea what's going on there.